app/src/containers/ListOfAdmins/index.js

```diff
@@ -52,14 +52,23 @@ class ListOfAdmins extends React.Component
       label: 'Last Access',
       minWidth: 170,
       align: 'center',
-      format: value => value.toLocaleString('fi-FI', { timeZone: 'UTC' })
+      format: value =>  {
+        if (value){
+          let time = new Date(value);
+          return +time.getDate()+ '.' +(time.getMonth()+1)+'.' +time.getFullYear()+' '+time.getHours()+'.'+time.getMinutes();
+        }
+        
+        }
     },
     {
-      id: 'userIsActivie',
+      id: 'isActive',
```

Comment:
Peter changed the backend data-structure recently. Hence update the property name here